mypy.ini

```diff
@@ -18,3 +18,6 @@ ignore_missing_imports = True
 
 [mypy-timezonefinder]
 ignore_missing_imports = True
+
+[mypy-constants]
+ignore_missing_imports = True
```

Comment:
I don't think this should be necessary

Reply:
mypy fails without it, should I add it back in?

Reply:
The point is that mypy should not be failing on that file. If it does, we need to figure out why and address the root cause and not simply ignore it.